libcontainer/rootfs_linux.go

```diff
@@ -518,7 +518,7 @@ func getCgroupMounts(m *configs.Mount) ([]*configs.Mount, error) {
 // dest is required to be an abs path and have any symlinks resolved before calling this function.
 //
 // if source is nil, don't stat the filesystem.  This is used for restore of a checkpoint.
-func checkProcMount(rootfs, dest, source string) error {
+func checkProcMount(rootfs, dest, source string, config *configs.Config) error {
```

Comment:
Again, this can be
```go
func (c *linucContainer) checkProcMount(rootfs, dest, source string) error {
config := c.config
```
or something similar

Reply:
See https://github.com/opencontainers/runc/pull/3224#discussion_r720606929